install/installation-source.md

```diff
@@ -22,6 +22,8 @@ Before you start, make sure you have installed:
     including downloads and instructions, see the
     [CMake documentation][cmake-download].
 *   C language compiler for your operating system, such as `gcc` or `clang`.
+*   Check the compatibility matrix[compatibility-matrix] of TimescaleDB versions
```

Comment:
Looks like we need the link here. Or I need to familiarize myself with this syntax.

```suggestion
*   Check the [compatibility matrix][compatibility-matrix] of TimescaleDB versions
```